File: 13_09_2024_Mirror_Tree.java
class Solution {
    // Function to convert a binary tree into its mirror tree.
    public void mirror(Node node) {
        if (node == null) {
            return;
        }
        
        mirror(node.left);
        mirror(node.right);
        
        Node temp = node.left;
        node.left = node.right;
        node.right = temp;
    }
}

File: 13_09_2024_Mirror_Tree.py
class Solution:
    # Function to convert a binary tree into its mirror tree.
    def mirror(self, node):
        if node is None:
            return
        
        self.mirror(node.left)
        self.mirror(node.right)
        
        temp = node.left
        node.left = node.right
        node.right = temp
        
        return

